src/number-input/stories/NumberInput.stories.tsx

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import { useForm, Controller, ControllerProps, Control } from "react-hook-form";
 import { Meta } from "@storybook/react";
```

Comment:
Make sure to arrange it in increasing stack order. 
```suggestion
import React from "react";
import { useForm, Controller } from "react-hook-form";
import { Meta } from "@storybook/react";
```